0000-template.md

```diff
@@ -1,5 +1,7 @@
 # Replace this with your awesome RFC title
 
+**Status:** `proposed`
```

Comment:
Can we enumerate all the possible values? Maybe: `draft`, `in-progress`, `approved`?

Reply:
Definitely.

My thinking had been that `proposed` made sense for the template since that's the starting state.

Also, the possible states are enumerates here: https://github.com/open-telemetry/rfcs#the-rfc-life-cycle

Reply:
Maybe add an italicized comment w/ the URL spelled out, for reviewing the lifecycle?

Reply:
Sorry I did not see all the definitions there. Looks good.